custodia/plugin.py

```diff
@@ -48,6 +48,11 @@ def __init__(self, message=None):
         super(CSStoreError, self).__init__(message)
 
 
+class CSStoreUnsupported(CustodiaException):
+    def __init__(self, message=None):
```

Comment:
You don't need ```__init__```. The exception class accepts a list of arguments. Please remove them from all store exceptions.